CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Change message log level from warning to debug when environment variables
   set inside a container or by APPTAINERENV have a different value than the
   environment variable on the host.
+- Fix a stage build hanging issue because of null TarOption.
```

Comment:
Describe that from a user's perspective, please:
```
- Fix hang when copying files between build stages while using suid mode without user namespaces.
```

Did you find any other situations where it hangs?  It's not clear how a null TarOptions would be related to suid mode without user namespaces, resulting in the use of fakeroot mode 4.

Reply:
> Describe that from a user's perspective, please:
> 
> ```
> - Fix hang when copying files between build stages while using suid mode without user namespaces.
> ```
> 
> Did you find any other situations where it hangs? It's not clear how a null TarOptions would be related to suid mode without user namespaces, resulting in the use of fakeroot mode 4.

thanks! haven't found other situations yet. it looks like the func CopyWithTarWithRoot has custom defined untar func which is used by third-party vendor library.